tests/Integration/SolrServer/CurlTest.php

```diff
@@ -22,6 +22,7 @@ public function setUp(): void
         // Set it to the PHP default of 13s.
         $adapter = new Curl();
         $adapter->setTimeout(CURLOPT_TIMEOUT);
+        $adapter->setConnectionTimeout(CURLOPT_CONNECTTIMEOUT);
```

Comment:
The value of `CURLOPT_CONNECTTIMEOUT` is `78`. That's a bit excessive for a connection timeout. The comment about the "PHP default of 13s" is misleading, that is not what `CURLOPT_TIMEOUT` having a value of `13` means. These values are merely a number by which each option is identified by libcurl internally.

It's probably better to set both options to a reasonable number. 5 seconds should be plenty for the connection timeout. A 15 second request timeout is definitely enough for the current integration tests as we're currently running them within 13 seconds.